pkgs/top-level/all-packages.nix

```diff
@@ -9181,6 +9181,8 @@ in
   pypy27Packages = pypy27.pkgs;
   pypy3Packages = pypy3.pkgs;
 
+  manylinuxPackages = callPackage ./../development/interpreters/python/manylinux { };
```

Comment:
This is kind of confusing as a top-level attribute (as it's python specific, but seems like it could be something broader), I think the `manylinux*` builders should live alongside buildPythonPackages under pythonX.pkgs

I'm also in favor of them renamed from `manylinux1Package` to `buildManylinux1Package`

Reply:
> This is kind of confusing as a top-level attribute (as it's python specific, but seems like it could be something broader)

These are not Python libraries, but just lists of derivations. They are used with Python packages but are not Python-version specific. Having some kind of indicator that they are related to Python is not be a bad idea.

> I'm also in favor of them being renamed from manylinuxXXXXPackage to buildManylinuxXXXXPackage

They are not build functions either. The lists are just that, lists, and the other ones are derivations but are not meant to be used; you do not want to patchelf libs via the symlink tree because you would retain too many dependencies that way.

Ideally every package set contains only derivations, not functions or anything else. We're far from there but its important to consider.

Reply:
hmmm, I abandon my renaming stance, but still think it should be moved to `python.manylinuxPackages`, or `python.pkgs.manylinuxPackages`